metadata-models/src/main/pegasus/com/linkedin/metadata/entity/DataFlowEntity.pdl

```diff
@@ -0,0 +1,30 @@
+namespace com.linkedin.metadata.entity
+
+import com.linkedin.common.DataFlowUrn
+
+/**
+ * Data model for a DataFlow entity
+ */
+record DataFlowEntity includes BaseEntity {
+
+  /**
+   * Urn for the DataFlow
+   */
+  urn: DataFlowUrn
+
+  /**
+   * Workflow orchestrator ex: Azkaban, Airflow
+   */
+  orchestrator: optional string
```

Comment:
we might want to consider making this an enum similar to Dataplatform (or include Azkaban/Airflow inside the Dataplatform enum)

Reply:
Yup, good point. I'd lean towards a new enum class, to reduce clutter in Dataplatform.

Reply:
Enums are great ... but they limit extensibility... if someone is running a company specific orchestrator and wants to ingest this in... they would have to fork the data model to create a new enum. So I would keep this as a string for now.

Reply:
I'm okay with moving to an urn for the orchestrator so that can help us with decorating with logos, etc in a declarative way. 

e.g.  "urn:li:orchestrator:airflow"   -> [ can lead to an about page about Airflow, logos etc].

Reply:
I implemented this urn now, and added an aspect with type and logoUrl.

Reply:
On further discussion, we decided to revert back to string, to avoid creating an extremely large PR.